### PR TITLE
jQuery speed

### PR DESCRIPTION
**Pull request by AurelioDeRosa**
Fixes gh-830

### CODE REVIEW

File: entries/jQuery.speed.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <title>jQuery.speed</title>
+  <desc>Creates an object containing a set of properties ready to be used in the definition of custom animations.</desc>
+  <signature>
+    <added>3.0.0</added>
+    <xi:include href="../includes/duration-argument.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
+    <xi:include href="../includes/easing-argument.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
+    <xi:include href="../includes/complete-argument.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
+  </signature>
+  <longdesc>
+    <p>The <code>$.speed</code> method provides a way to define properties, such as <code>duration</code>, <code>easing</code>, and <code>queue</code>, to use in a custom animation. By using it, you don't have to implement the logic that deals with default values and optional parameters.</p>
```
Comment (scottgonzalez) [inline entries/jQuery.speed.xml]: Add parens to `$.speed`. Same on next line.

File: entries/jQuery.speed.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<entry type="method" name="jQuery.speed" return="PlainObject">
+  <title>jQuery.speed</title>
+  <desc>Creates an object containing a set of properties ready to be used in the definition of custom animations.</desc>
+  <signature>
+    <added>3.0.0</added>
```
Comment (scottgonzalez) [inline entries/jQuery.speed.xml]: This has existed for a long time.

Comment (AurelioDeRosa) [inline entries/jQuery.speed.xml]: Can you give me the version?

Comment (scottgonzalez) [inline entries/jQuery.speed.xml]: Looks like the three parameter version was added in 1.1 and a 2 parameter version existed in 1.0.

File: entries/jQuery.speed.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<entry type="method" name="jQuery.speed" return="PlainObject">
+  <title>jQuery.speed</title>
+  <desc>Creates an object containing a set of properties ready to be used in the definition of custom animations.</desc>
+  <signature>
```
Comment (scottgonzalez) [inline entries/jQuery.speed.xml]: There are multiple signatures. You can pass an object instead of individual parameters.

Comment (AurelioDeRosa) [inline entries/jQuery.speed.xml]: Thank you for the review @scottgonzalez. I'll update the PR soon.

**Comment by AurelioDeRosa** [comment]
PR updated @scottgonzalez. Please review it when you have time.

**Comment by scottgonzalez** [comment]
Looks good.

**Comment by AurelioDeRosa** [comment]
**Note**: I've merged this PR in master since the method was implemented and exposed since version 1.0

**Pull request closed without merging**